R/tm_variable_browser.R

```diff
@@ -498,24 +490,10 @@ srv_variable_browser <- function(id,
       )
     })
 
-    ### REPORTER
-    if (with_reporter) {
-      card_fun <- function(comment) {
-        card <- teal::TealReportCard$new()
-        card$set_name("Variable Browser Plot")
-        card$append_text("Variable Browser Plot", "header2")
-        if (with_filter) card$append_fs(filter_panel_api$get_filter_state())
-        card$append_text("Plot", "header3")
-        card$append_plot(variable_plot_r(), dim = pws$dim())
-        if (!comment == "") {
-          card$append_text("Comment", "header3")
-          card$append_text(comment)
-        }
-        card
-      }
-      teal.reporter::add_card_button_srv("add_reporter", reporter = reporter, card_fun = card_fun)
-    }
-    ###
+    reactive({
+      validation_checks()
+      teal.data::teal_data(plot = variable_plot_r()) |> teal.code::eval_code("plot")
```

Comment:
Maybe let's just show the plot without any code to reproduce it, wdyt?

```suggestion
      qenv <- teal.data::teal_data(plot = variable_plot_r()) |> teal.code::eval_code("plot")
      teal_card(qenv)[length(teal_card(qenv))]
```